src/components/ContactInfoSummary.tsx

```diff
@@ -0,0 +1,228 @@
+import { Col, Row, Space, Typography } from 'antd';
+import dateformat from 'dateformat';
+import React from 'react';
+import styled from 'styled-components';
+import { ContactInfo } from '../containers/setContacts/ducks/types';
+import { DEFAULT_IMAGE } from '../utils/copy';
+const { Text } = Typography;
+
+interface ContactInfoSummaryProps {
+  userId: number;
+  info: ContactInfo;
+}
+
+const ContactRow = styled.div`
+  height: 200px;
+  margin-bottom: 16px;
+`;
+
+const ProfilePicture = styled.img`
+  width: 200px;
+  height: 200px;
+
+  object-fit: cover;
+  float: left;
+  margin-right: 16px;
+`;
+
+const CopyContainer = styled.div`
+  width: 100%;
+  height: 200px;
+`;
+
+const NameTitle = styled(Text)`
+  font-size: 1.6rem;
+  display: block;
+  margin-bottom: 16px;
+`;
+
+const SectionTitle = styled(Text)`
+  font-weight: 800;
+  font-size: 1.2rem;
+  display: block;
+  margin-bottom: 16px;
+`;
+
+const FamilyDetails: React.FC<ContactInfoSummaryProps> = ({ userId, info }) => {
+  return (
+    <>
+      <SectionTitle>Account Owner</SectionTitle>
```

Comment:
There seems to be a lot of repeated code here that could be broken down in various ways. 200 lines seems a bit long for a component. I think we should have a component for displaying a single contact/child, as well as a component for displaying the user's account info. These may even both consist of a more generic component but I'm not sure that's necessary.

Reply:
I agree with this. You could have a ContactRow component with `email?` to reduce a lot of the repetition between parent and children contacts.

Reply:
+1